executor.go

```diff
@@ -1252,6 +1252,7 @@ func (e *Executor) executeSetColumnAttrs(ctx context.Context, index string, c *p
 	// Copy args and remove reserved fields.
 	attrs := pql.CopyArgs(c.Args)
 	delete(attrs, colName)
+	delete(attrs, "frame")
```

Comment:
Add a comment in the code here:
While `frame` could technically work as a ColumnAttr argument, we are treating it as a reserved word
primarily to avoid confusion. Also, if we ever need to make ColumnAttrs frame-specific, then having
this reserved word prevents backward incompatibility.